design/templates/solution_design.md

```diff
@@ -0,0 +1,71 @@
+# Solution Design - Template
+[//]: # (Change the title above from "Template" to your design's title) 
```

Comment:
was is the template for the solution design added with this PR?

Reply:
could be a sign that a rebase is needed?